projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts

```diff
@@ -52,6 +52,12 @@ export class DataLayerManagerService {
   enabledLayers$ = this.selectedLayers$.pipe(map((layers) => layers.filter((layer) => layer.enabled)));
   loading$ = new BehaviorSubject<boolean>(false);
 
+  lastBPLayers$ = this.allLayers$.pipe(
+    map((layers) => layers.filter((layer) => layer.name === 'Blood Pressure')),
+    map((layers) => layers.map((layer) => layer.datasets.map((data) => data.data))),
+    map((layers) => layers.map((layer) => layer.map((data) => data.slice(-1))))
+  );
```

Comment:
This should probably be in the app, not the DataLayerManager. Also, I think this could be simplified to use a single rxjs map operator.